src/software/thunderscope/colors.py

```diff
@@ -1,11 +1,14 @@
 from pyqtgraph.Qt import QtCore, QtGui
 
 BLUE_ROBOT = QtGui.QColor(255, 100, 0, 255)
-FIELD_COLOR = QtCore.Qt.white
-FIELD_LINE_COLOR = QtCore.Qt.white
+FIELD_COLOR = "w"
```

Comment:
I feel like these would be better as enums if possible but that's a *super* nit.

Reply:
agreed, changed

itll also be nicer when we have themes